app/apps/camunda/services.py

```diff
@@ -35,43 +35,50 @@ def get_process_definitions(self):
         processes = []
         response = self._process_request("process-definition")
 
-        if response.ok:
```

Comment:
De tests faalden hierop (ok isn't available in response ofzo iets, dus ik moest dit herschrijven naar een ty catch)

Reply:
Denk dat het dan beter is om op `if not response.raise_on_status` te checken dan een generieke exception handler

Reply:
Done. Ik check nu op een RequestException in plaats van een generieke exception